.gitignore

```diff
@@ -161,5 +161,5 @@ cython_debug/
 
 # Ignore generated data
 chicago/files_for_*
-chicago/chicago_pin_universe.csv
+chicago/chicago_pin_universe*.csv
```

Comment:
Changing up the cache to index by year, since the PIN universe can now change depending on the values of `start_date` and `end_date`.